docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+
+```
+docker push <region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>:<tag>
+```
+
+3. Create the pull secret for the remote Docker registry:
+
+```
+ kubectl -n <domain namespace> create secret docker-registry <secret name> \
+     --docker-server=<region-key>.ocir.io/<tenancy-namespace>/<repo-name> \
+     --docker-username=your.email@some.com \
+     --docker-password=your-password \
+     --docker-email=your.email@some.com
+
+```
+
+4. Update the domain template file `$SAMPLEDIR/k8s-domain.yaml.template` to provide the `imagePullSecrets`:
+
+```
+  imagePullSecrets:
+  - name: <secret name>
+
+```
+
+5. Export the environment variables for the image name and tag using the same values in step 1:
+
+```
+export MODEL_IMAGE_NAME="<region-key>.ocir.io/<tenancy-namespace>/<repo-name>/<image-name>"
+export MODEL_IMAGE_TAG="<tag>"
+```
+
+### Create and deploy your Kubernetes resources
+
+To deploy the sample operator domain and its required Kubernetes resources, use the sample script, `$SAMPLEDIR/run_domain.sh`, which will perform the following steps for you:
+
+  - Deletes the domain with a `DomainUID` of `domain1` in the namespace, `sample-domain1-ns`, if it already exists.
+  - Creates a secret containing your WebLogic administrator user name and password.
+  - Creates a secret containing your Model in Image runtime encryption password:
+    - All model-in-image domains must supply a runtime encryption secret with a `password` value.
+    - It is used to encrypt configuration that is passed around internally by the Operator.
+    - The value must be kept private but can be arbitrary: you can optionally supply a different secret value every time you restart the domain.
+  - Creates secrets containing your RCU access URL, credentials, and prefix (these are unused unless the domain type is `JRF`).
+  - Creates a config map containing an additional WDT model properties file, `$SAMPLEDIR/model1.20.properties`.
+  - Generates a domain resource YAML file, `$WORKDIR/k8s-domain.yaml`, using `$SAMPLEDIR/k8s-domain.yaml.template`.
+  - Deploys `k8s-domain.yaml`.
+  - Displays the status of the domain pods.
+
+The script expects `WDT_DOMAIN_TYPE` and `WORKDIR` to already be initialized. (See [Prerequisites for all domain types](#prerequisites-for-all-domain-types).)
+
+Run the script:
+
+  ```
+  $SAMPLEDIR/run_domain.sh
+  ```
+
+At the end, you will see the message `Getting pod status - ctrl-c when all is running and ready to exit`. Then you should see a WebLogic Administration Server and two Managed Server pods start. After all the pods are up, you can use `ctrl-c` to exit the build script.
+
+
+### Optionally test the sample application
+
+1. Ensure Traefik has been installed and is servicing external port 30305, as per [Prerequisites for all domain types](#prerequisites-for-all-domain-types).
+
+2. Create a Kubernetes Ingress for the domain's WebLogic cluster in the domain's namespace by using the sample Helm chart:
+
+   For Helm 2.x:
```

Comment:
We're not going to support Helm 2.x with operator 3.0.0.

Reply:
Tom AI: Ok.

Reply:
Done.